configs/main.go

```diff
@@ -94,21 +96,18 @@ func New() *Configs {
 	}
 	projectViper := viper.New()
 
-	// NOTE: viper.ConfigFileNotFound not produced if using projectViper.setConfigFile()
-	projectViper.AddConfigPath(projectDir)
-	projectViper.SetConfigName("config")
-	projectViper.SetConfigType("json")
+	projectPath := path.Join(projectDir, "./config.json")
+	projectViper.SetConfigFile(projectPath)
 	err = projectViper.ReadInConfig()
-	if _, ok := err.(viper.ConfigFileNotFoundError); ok {
-		// Config file not found; that's okay
+	if os.IsNotExist(err) {
+		// That's okay, continue
 	} else if err != nil {
-		// Config file was found but another error was produced
-		fmt.Printf("Unable to load project config! %#v\n", err)
+		fmt.Println("Unable to load project config!")
 	}
 
 	projectConfig := &Config{
 		viper:      projectViper,
-		configPath: projectViper.ConfigFileUsed(),
```

Comment:
This was the crux of the issue. If a config wasn't found, this path would be empty.